src/quantile.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-unused-vars */
+/**
+ * @param {object} data - x sorted in increasing order and y the number of occurrence of each x value
+ * @param {numbers} k, q - the kth q-quantile, thus k should be less than or equal to q, should be positive integers
```

Comment:
Why not only one parameters between 0,1 like in mathlab ? https://www.mathworks.com/help/stats/quantile.html
Please also use full name for variables and not just 'p'

Reply:
Good idea, I have changed to cumProb (cumulative probability)